src/services/user/useOwner.ts

```diff
@@ -45,6 +47,15 @@ export function useOwner({
     queryKey: ['owner', variables, provider, query],
     queryFn: ({ signal }) =>
       Api.graphql({ provider, query, variables, signal }).then((res) => {
+        const parsedData = DetailOwnerSchema.safeParse(res)
+        if (!parsedData.success) {
+          return Promise.reject({
+            status: 404,
+            data: {},
+            dev: 'useOwner - 404 failed to parse',
+          } satisfies NetworkErrorObject)
+        }
+
         return res?.data?.owner
```

Comment:
We'll need to return the data from the `parsedData` to get the types out from the zod schema.

Reply:
good catch, fixed

Reply:
Actually, it became kinda insane - I'm going to put the `useOwner` and org upload related stuff in new prs for this spun-off ticket - https://github.com/codecov/engineering-team/issues/2857